frontend/packages/console-shared/src/test-utils/utils.ts

```diff
@@ -157,3 +157,25 @@ export function searchYAML(needle: string, name: string, namespace: string, kind
   const result = execSync(`kubectl get -o yaml -n ${namespace} ${kind} ${name}`).toString();
   return result.search(needle) >= 0;
 }
+
+type CardInfo = {
+  text: string;
+  index: number;
+};
+
+const getOperatorHubCards = async (): Promise<CardInfo[]> => {
+  const selector = $$('article .co-resource-item__resource-name');
+  const cards: CardInfo[] = await selector.map(async (elem, index) => {
+    return {
+      text: await elem.getText(),
+      index,
+    };
+  });
+  return cards;
+};
+
+export const getOperatorHubCardIndex = async (name: string): Promise<CardInfo['index']> => {
+  const cards = await getOperatorHubCards();
+  const reqdCard = cards.find((elem) => elem.text === name);
```

Comment:
what if the name is not present? Did you handle the case?

Reply:
I don't think we should worry about that in this function. Maybe run this function only after verifying that those elements indeed exist.

Reply:
P.S It just fails.